mmv1/third_party/terraform/services/lustre/resource_lustre_instance_test.go

```diff
@@ -0,0 +1,111 @@
+package lustre_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/plancheck"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccLustreInstance_update(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"network_name":  acctest.BootstrapSharedTestNetwork(t, "default-vpc"),
```

Comment:
Same here, should be unique per test

Reply:
Spoke offline we will rename this in follow up CL as this requires quota changes.